index.html

```diff
@@ -307,6 +307,70 @@ <h3 id="type-links">External references should use typed term</h3>
           }
         </pre>
       </section>
+
+      <section>
+        <h3 id="context-order">Ordering of array elements</h3>
+        <p class="practicedesc">By default, arrays in <a class="externalDFN" href="https://www.w3.org/TR/json-ld/#sets-and-lists">JSON-LD do not convey any ordering of contained elements</a>.
```

Comment:
I think the link makes more sense with a bit more labeling, but YMMV.
```suggestion
        <p class="practicedesc">By default, <a class="externalDFN" href="https://www.w3.org/TR/json-ld/#sets-and-lists">arrays in JSON-LD do not convey any ordering of contained elements</a>.
```